src/Goteo/Application/Lang.php

```diff
@@ -112,14 +112,6 @@ static public function trans($id, array $parameters = array(), $locale = null) {
         return static::$translator->trans($id, $parameters, null, $locale);
     }
 
-    /**
-     * Handy method for the transChoice() function of the Symfony Component Translator
-     */
-    static public function transChoice($id, $number, array $parameters = array(), $domain = null, $locale = null) {
-        static::factory();
-        return static::$translator->transChoice($id, $number, $parameters, $domain, $locale);
-    }
```

Comment:
I've already removed this method (it was supposed to be done in the Symfony 4.2 upgrade), since is not being used.